include/gatekeeper_ggu.h

```diff
@@ -232,6 +232,7 @@ struct ggu_bpf {
 struct ggu_policy {
 	uint8_t state;
 	struct ip_flow flow;
+	uint32_t flow_hash_val;
```

Comment:
`struct ggu_policy` is meant to just carry a policy decision and the hash value of the flow is not part of it. The hash value only makes sense to GK blocks. This is relevant because `struct ggu_policy` is used throughout the code of Gatekeeper and the hash value doesn't have a meaning elsewhere. For example, although `gt/main.c:lookup_policy_decision()` initializes a `struct ggu_policy`, it does not need to set the hash value, or even would be able to do so.

The solution is to modify `struct gk_cmd_entry`, the only struct that really has a meaning for the hash value.